src/components/PharmacyList/PharmacyList.tsx

```diff
@@ -1,61 +1,69 @@
-import { PharmacyContainer } from "../PharmacyContainer";
+import { PharmacyCard } from "../PharmacyCard";
 import React from "react";
+import { useListVaccineAvailabilityApiV1VaccineAvailabilityGet } from "../../apiClient";
 
-interface Pharmacy {
-  id: string;
-  pharmacyName: string;
-  booking: boolean;
-  address: string;
-  phone: string;
-  website: string;
-  lastUpdated: string;
-}
+type PharmacyProps = React.ComponentProps<typeof PharmacyCard>;
 
 export function PharmacyList() {
```

Comment:
Is this a good candidate to be a container? Might be useful to build out a `containers` and maybe even `routes` folder. I think there was also some tasks in the future to have an FAQ page so could be useful to scaffold out the foundation for this pattern. It may be out of the scope of this PR though.

Reply:
Yes, was thinking the same thing, but yes, out of scope... this ticket was just to provide guidance about consuming an endpoint lol